src/sentry/static/sentry/app/views/usageStats/types.tsx

```diff
@@ -1,7 +1,25 @@
-import {MinimalProject} from 'app/types';
+export enum Outcome {
+  ACCEPTED = 'accepted',
+  FILTERED = 'filtered',
+  INVALID = 'invalid',
+  DROPPED = 'dropped',
+}
+
+/**
+ * Raw response from API endpoint
+ */
+export type UsageSeries = {
```

Comment:
confused how the PRs end up merging together -- seems like this is newly defined compared to the one in org-stats-5